doc/use-on-icub-head.md

```diff
@@ -0,0 +1,62 @@
+
+**Warning: if your iCub robot is not currently using the robotology-superbuild, do not attempt to migrate it
+without coordinating with the iCub support.  For any doubt, please open an issue at https://github.com/robotology/icub-support.**
+
+## Prepare your system 
+The operating system contained in the `icub-head`/`pc104` is tipically installed by IIT, for more information on it, please
+check the relevant documentation at http://wiki.icub.org/wiki/The_Linux_on_the_pc104 . On this machine, 
+in `/home/icub/.bashrc_iCub` a script containing several enviroment variables definitions is provided. If you want to  migrate to use
+the robotology-superbuild, you need to remove the existing `.bashrc_iCub` file, and substitute it with the one provided in https://git.robotology.eu/MBrunettini/icub-environment/raw/master/home/bashrc_iCub_superbuild .
+After you modified the `.bashrc_iCub` script, reboot the computer and  in a new shell check che values of the `ROBOTOLOGY_SUPERBUILD_SOURCE_DIR` and `YARP_ROBOT_NAME` env variables.
+If `ROBOTOLOGY_SUPERBUILD_SOURCE_DIR` contains `/usr/local/src/robot/robotology-superbuild`, and `YARP_ROBOT_NAME` contains the string specific to your robot,
+then the modification of the `.bashrc_iCub` was succesful. 
+
+## Get software source code, compile and install
+On the `icub-head`/`pc104`, the software repository necessary to run the iCub robot are contained in the `/usr/local/src/robot` directory.
+On a new machine, this directory should be empty. All the relevant software can download and build with the following commands:
+~~~sh
+cd /usr/local/src/robot
+git clone https://github.com/robotology/robotology-superbuild.git
+cd robotology-superbuild
+mkdir build
+cmake -DROBOTOLOGY_USES_GAZEBO:BOOL=OFF -DROBOTOLOGY_ENABLE_ICUB_HEAD:BOOL=ON ..
+make
+~~~
+The main difference over the standard installation of the robotology-superbuild, is that the `ROBOTOLOGY_USES_GAZEBO` option
+is disabled (as the Gazebo is tipically not installed in the `icub-head`/`pc104` machine) and the `ROBOTOLOGY_ENABLE_ICUB_HEAD` option
+is enabled, to enable all the YARP devices that are necessary to interface with the internal communication bus of the iCub robot.
+The `make` command in this case will download, compile and install all the software necessary to run the robot itself.
+
+If you need to use other *superbuild profiles*, they can be easily enabled with the corresponding CMake option.
+
+**Important: do not run `make install` for the superbuild: the superbuild already installs all the software in 
+`/usr/local/src/robot/robotology-superbuild/build/install`, and installing the software elsewhere on the robot is not currently supported.**
+
+**Important: If you are using an old iCub that still uses the CAN internal bus, you also need to enable the `ROBOTOLOGY_USES_CFW2CAN` CMake options. 
+See the main robotology-superbuild README for more detailed information.**
```

Comment:
I would add those details here since they are specific to building the superbuild in the iCub Head.

Reply:
I would prefer to avoid duplication of the content itself. I can definitely improve the usability by adding a link here to the specific section of the README, or move the content here and have in the README a link here. Which option do you prefer? I am almost 100% sure that if I duplicate the content I know that it will be soon inconsistent, as it will be changed in the README and not here. : )